static-web/index.html

```diff
@@ -0,0 +1,230 @@
+<!-- Instructions
```

Comment:
Never put a comment before the doctype. The browser needs to know it's dealing with HTML before anything else.